src/python/grpcio_health_checking/grpc/health/v1alpha/health.py

```diff
@@ -0,0 +1,73 @@
+import abc
+import enum
+from grpc.health.v1alpha import health_pb2
+
+
+@enum.unique
+class HealthStatus(enum.Enum):
+  UNKNOWN = health_pb2.HealthCheckResponse.UNKNOWN
+  SERVING = health_pb2.HealthCheckResponse.SERVING
+  NOT_SERVING = health_pb2.HealthCheckResponse.NOT_SERVING
+
+
+class _HealthServicer(health_pb2.EarlyAdopterHealthServicer):
+
+  def __init__(self):
+    self._server_status = {}
+
+  def Check(self, request, context):
+    server_key = (request.host, request.service,)
```

Comment:
I don't think this local field justifies its existence - can you just write

return health_pb2.HealthCheckResponse(
    status=self._server_status[request.host, request.service].value)

on the next line?